src/styles/containers/About.scss

```diff
@@ -2,15 +2,16 @@
 
 :global {
 	.About {
-		padding: 60px 60px 0;
+		height: 100%;
```

Comment:
Just center this rather than place static padding values that may not account for a number of variables.